apps/openmw/mwmechanics/summoning.hpp

```diff
@@ -22,7 +22,7 @@ namespace MWMechanics
                             float magnitude, float remainingTime = -1, float totalTime = -1);
 
         /// To call after all effect sources have been visited
-        void process();
+        void process(bool cleanup);
```

Comment:
I don't think we really need documentation for a function that's not used (or not going to be used) more than once.